src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -84,6 +84,7 @@
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
 import org.codehaus.jackson.type.TypeReference;
+import static org.apache.commons.httpclient.util.URIUtil.encodePath;
```

Comment:
this method is removed in newer releases of httpclient 😒

Reply:
Yep, there was a reason I had the `split -> Util.rawEncode -> join` in the code that was copied. We need to either switch to that (what I recommend for this PR) or replace all the URL construction with an actual solution such as https://github.com/damnhandy/Handy-URI-Templates (what I recommend as the long term direction for this plugin)

Reply:
Handy URI Templates 👍 💜

Reply:
@casz yep, but I think switching to that is Out Of Scope for this PR... a separate PR yes, but not this one ;-)

Reply:
Done